pkg/workflows/sdk/compute_generated.go

```diff
@@ -76,6 +76,58 @@ func Compute1[I0 any, O any](w *WorkflowSpecFactory, ref string, input Compute1I
 	return &computeOutputCap[O]{(&Step[ComputeOutput[O]]{Definition: def}).AddTo(w)}
 }
 
+func Compute1WithMetadata[I0 any, O any](w *WorkflowSpecFactory, ref string, input Compute1Inputs[I0], compute func(Runtime, I0, capabilities.RequestMetadata) (O, error)) ComputeOutputCap[O] {
```

Comment:
was this auto generated?

Reply:
Seems like it. @RyanRHall  Could you advice on whether this change is needed and if so, how to re-generate this code after recent changes in this PR?

Reply:
@nolag (not Ryan Hall)

Reply:
Thanks for the right tag :)

Reply:
This is generated code.  The generator lives in the folder gen in the same directory.  My understanding is that we're only using this to hack consensus and that it's not something we want to check into develop.  Once we have read consensus, we won't use this.  I vote against merging this change.

Reply:
As discussed offline, I've added this function to generator and added a comment specifying that this function is temporary and not intended to be used outside testing. This function would be removed once more tooling is available for testing.